static/js-src/feature-page.js

```diff
@@ -36,13 +37,17 @@ const shareFeature = () => {
 };
 
 function copyURLToClipboard() {
-  event.preventDefault();
```

Comment:
should this be removed?

Reply:
Good point.  It is needed.  The way that that line was written looked like a defect to me because `event` is not defined anywhere.  I rewrote calls to `preventDefault()` for each of the listeners and tested them.